docs/modules/ROOT/pages/how-tos/recover-from-backup.adoc

```diff
@@ -70,15 +70,15 @@ vault login -method=oidc
 +
 [source,console]
 ----
-export RESTIC_REPOSITORY=s3:$(yq read -cjd'*' catalog/manifests/cluster-backup/05_schedule.yaml | jq -r '.[] | select(.kind == "Schedule") | .spec.backend.s3 | .endpoint + "/" + .bucket')
+export RESTIC_REPOSITORY=s3:$(yq -o=json '.' catalog/manifests/cluster-backup/10_object.yaml | jq -r 'select(.kind == "Schedule")| .spec.backend.s3 | .endpoint + "/" + .bucket')
```

Comment:
If we're depending on yq v4, we can drop the `jq` completely:

```suggestion
export RESTIC_REPOSITORY=s3:$(yq -o=json 'select(.kind == "Schedule")| .spec.backend.s3 | .endpoint + "/" + .bucket' catalog/manifests/cluster-backup/10_object.yaml)
```

Reply:
be my guest, hadn't had enough coffee for further refactoring 😅